composable-ui/package.json

```diff
@@ -21,6 +21,7 @@
     "@chakra-ui/theme-tools": "^2.0.16",
     "@composable/cms-generic": "workspace:*",
     "@composable/commerce-generic": "workspace:*",
+    "@composable/klevu": "workspace:*",
```

Comment:
@rallu please remove this modification of  the composable-ui `package.json` and provide this step in an installation step of the Klevu package:

`pnpm add @composable/klevu`